src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -325,6 +325,7 @@ private void OnUpdateGraphCompleted(AsyncTask task)
 
             // Notify listeners (optional) of completion.
             RunSettings.RunEnabled = true; // Re-enable 'Run' button.
+            Notifications.Instance.PostNotification("Run complete.", NotificationLevel.Mild);
```

Comment:
What if there are multiple home workspaces?  The notification needs to somehow tell the callee about which workspace.